app/controllers/companies_controller.rb

```diff
@@ -38,6 +40,7 @@ def edit
   def create
     authorize Company
     @company = Company.new(company_params)
+    @company.addresses.first.addressable = @company  # not sure why Rails doesn't assign this automatically
```

Comment:
I couldn't make that work.  I wonder if it has something to do with the fact that we're dealing with an Array of Addresses for a Company?